add-missing-campaign.js

```diff
@@ -0,0 +1,207 @@
+export const addMissingCampaign = async (client) => {
```

Comment:
I like the idea of using closure to provide V% client 👍 Please note that only the `addValidationRulesAndCampaign` function needs that context. The `createRewardPromotionObject`, `firstValidationRule`, `secondValidationRule` and `thirdValidationRule` functions do not need this context so they can be put outside of the outer function.